tajo-storage/tajo-storage-common/src/main/java/org/apache/tajo/storage/StorageManager.java

```diff
@@ -538,7 +538,8 @@ public Appender getAppender(OverridableConf queryContext,
     if (scannerClass == null) {
       scannerClass = conf.getClass(
           String.format("tajo.storage.scanner-handler.%s.class", handlerName), null, Scanner.class);
-      SCANNER_HANDLER_CACHE.put(handlerName, scannerClass);
+      if (scannerClass != null)
```

Comment:
@blrunner Thank you for your review! I hope everything goes well. Please comment me anything to be changed.